### ISSUE TITLE
Password-protect read access to wallets

### ISSUE THREAD

**Issue by leafcutterant** [darkwallet/darkwallet#47]
One has no option to log out of the Dark Wallet. Not even by exiting the browser. There should be something like this to make it secure.

**Comment by tilthz** [comment]
You are right. But there is no point on having a logout if you can't password-protect the read access to a wallet, so it'll have to wait until that gets implemented.

**Comment by leafcutterant** [comment]
True.
